### DISCUSSION TITLE
Talk not working. Do I miss something?

### DISCUSSION THREAD

**Discussion by jpameli** [nextcloud/all-in-one#763] [Questions] [answered]
Hi! I just installed a fresh Ubuntu 22.04 on an VPS. After installing docker and setting up the nginx reverse proxy I deployed the docker container like this:

sudo docker run -it --name nextcloud-aio-mastercontainer --restart always -p 8080:8080 -e APACHE_PORT=11000 --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config --volume /var/run/docker.sock:/var/run/docker.sock:ro nextcloud/all-in-one:latest

AIO shows, that all containers are running (green). Also everything seems to work great. I just have a problem with talk. When I open talk I get a message "Error establishing the signaling connection / german: Fehler beim Aufbau der Signaling-Verbindung".

I did not set up ufw, so I think this shouldn't be a firewall issue. Anything else that I can check to get talk running?

**Comment by jpameli** [comment, marked as answer]
I was able to fix this by adding this to the vhost:

    proxy_cache_bypass  $http_upgrade;
    proxy_set_header Upgrade           $http_upgrade;
    proxy_set_header Connection        "upgrade";
    proxy_set_header Host              $host;
    proxy_set_header X-Real-IP         $remote_addr;
    proxy_set_header X-Forwarded-For   $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_set_header X-Forwarded-Host  $host;
    proxy_set_header X-Forwarded-Port  $server_port;

:)

**Reply by szaimen** [reply]
Hello, can you create a PR that adds this to the nginx docs if you think these changes are necessary?
